test/common/sidechain.go

```diff
@@ -73,11 +73,12 @@ func AddCandidateWithStake(t *testing.T, transactor *transactor.Transactor,
 	err = InitializeCandidate(auth, sgnAddr, minAmt)
 	ChkTestErr(t, err, "failed to initialize candidate")
 
+	// bug here
```

Comment:
what is the bug?

Reply:
It was just a debug note. I forgot to delete it.

Reply:
deleted